lib/app/sequence_base.rb

```diff
@@ -813,6 +813,22 @@ def date_comparator_value(comparator, date)
           ''
         end
       end
+
+      def fetch_all_search_results(bundle)
+        resources = bundle&.entry&.map { |entry| entry&.resource }
+        page_count = 1
+        next_bundle_link = bundle&.link&.find { |link| link.relation == 'next' }&.url
+        next_bundle = bundle.next_bundle
```

Comment:
The code duplication in this method should be eliminated.

Reply:
Removed duplicated code.